app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectedJenkinsServers.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import moment from 'moment';
+import '@testing-library/jest-dom/extend-expect';
+import { ConnectedJenkinsServers, timeFromNow } from './ConnectedJenkinsServers';
+import { EventType, JenkinsServer } from '../../../../src/common/types';
+
+describe('timeFromNow util', () => {
```

Comment:
Thought about setting this up in its own test file but figured it made more sense to go here at this point as it is only being used by this component. If we end up using it elsewhere we can move both the util function and its tests to their own files.